wgsl/index.bs

```diff
@@ -635,42 +768,68 @@ TODO: the RGBA8UNORM-SRGB format is more complicated than this description.
 * type must be `f32`, `i32` or `u32`
 
 ### Read-only Storage Texture Types ### {#texture-ro}
+
+A read-only storage texture supports reading a single texel, with automatic conversion
+of the stored texel value to a usable shader value.
+A read-only storage texture type is parameterized by a [=texel format=], which in turn
+determines the conversion function as specified in [[#texel-formats]].
+
 <pre class='def'>
 `texture_ro_1d<texel_format>`
-  %1 = OpTypeImage type(texel_format) 1D 0 0 0 2 texel_format ReadOnly
+  # %1 = OpTypeImage type(texel_format) 1D 0 0 0 2 texel_format ReadOnly
 
 `texture_ro_1d_array<texel_format>`
-  %1 = OpTypeImage type(texel_format) 1D 0 1 0 2 texel_format ReadOnly
+  # %1 = OpTypeImage type(texel_format) 1D 0 1 0 2 texel_format ReadOnly
 
 `texture_ro_2d<texel_format>`
-  %1 = OpTypeImage type(texel_format) 0 0 0 2 texel_format ReadOnly
+  # %1 = OpTypeImage type(texel_format) 0 0 0 2 texel_format ReadOnly
 
 `texture_ro_2d_array<texel_format>`
-  %1 = OpTypeImage type(texel_format) 0 1 0 2 texel_format ReadOnly
+  # %1 = OpTypeImage type(texel_format) 0 1 0 2 texel_format ReadOnly
 
 `texture_ro_3d<texel_format>`
-  %1 = OpTypeImage type(texel_format) 3D 0 0 0 2 texel_format ReadOnly
+  # %1 = OpTypeImage type(texel_format) 3D 0 0 0 2 texel_format ReadOnly
 </pre>
-* type(texel_format) is one of `f32`, `i32`, or `u32`, depending on the storage type.
+
+In the SPIR-V mapping:
+* the *Image Format* parameter of the image type declaration is
+    as specified by the SPIR-V texel format correspondence table in [[#texel-formats]].
+* the *Sampled Type* parameter of the image type declaration is
+    the SPIR-V scalar type corresponding to the channel format for the texel format.
 
 ### Write-only Storage Texture Types ### {#texture-wo}
+
+A write-only storage texture supports writing a single texel, with automatic conversion
+of the shader value to a stored texel value.
+A write-only storage texture type is parameterized by a [=texel format=], which in turn
+determines the conversion function as specified in [[#texel-formats]].
+The *inverse* of the transfer function is used to convert the shader value to
```

Comment:
it says "conversion function" and then "transfer function". Is this the same thing?